lua/lvim/core/which-key.lua

```diff
@@ -181,17 +186,16 @@ M.config = function()
       g = {
         name = "Git",
         g = { "<cmd>lua require 'lvim.core.terminal'.lazygit_toggle()<cr>", "Lazygit" },
-        j = { "<cmd>lua require 'gitsigns'.next_hunk({navigation_message = false})<cr>", "Next Hunk" },
-        k = { "<cmd>lua require 'gitsigns'.prev_hunk({navigation_message = false})<cr>", "Prev Hunk" },
-        l = { "<cmd>lua require 'gitsigns'.blame_line()<cr>", "Blame" },
-        p = { "<cmd>lua require 'gitsigns'.preview_hunk()<cr>", "Preview Hunk" },
-        r = { "<cmd>lua require 'gitsigns'.reset_hunk()<cr>", "Reset Hunk" },
-        R = { "<cmd>lua require 'gitsigns'.reset_buffer()<cr>", "Reset Buffer" },
-        s = { "<cmd>lua require 'gitsigns'.stage_hunk()<cr>", "Stage Hunk" },
-        u = {
-          "<cmd>lua require 'gitsigns'.undo_stage_hunk()<cr>",
-          "Undo Stage Hunk",
-        },
+
+        j = { "<cmd>Gitsigns next_hunk<cr>", "Next Hunk" },
+        k = { "<cmd>Gitsigns prev_hunk<cr>", "Prev Hunk" },
+        l = { "<cmd>Gitsigns blame_line<cr>", "Blame Line" },
+        L = { "<cmd>lua require 'gitsigns'.blame_line({full=true})<cr>", "Blame Line (full)" },
+        p = { "<cmd>Gitsigns preview_hunk<cr>", "Preview Hunk" },
+        r = { "<cmd>Gitsigns reset_hunk<cr>", "Reset Hunk" },
+        s = { "<cmd>Gitsigns stage_hunk<cr>", "Stage Hunk" },
+        u = { "<cmd>Gitsigns undo_stage_hunk<cr>", "Undo Stage Hunk" },
+        R = { "<cmd>Gitsigns reset_buffer<cr>", "Reset Buffer" },
```

Comment:
why do we need to use `Gitsigns` in some instances? can't we use `require 'gitsigns'` everywhere?

like the example from their own repo

```lua
   -- Actions
    map('n', '<leader>hs', gitsigns.stage_hunk)
    map('n', '<leader>hr', gitsigns.reset_hunk)
    map('v', '<leader>hs', function() gitsigns.stage_hunk {vim.fn.line('.'), vim.fn.line('v')} end)
    map('v', '<leader>hr', function() gitsigns.reset_hunk {vim.fn.line('.'), vim.fn.line('v')} end)
    map('n', '<leader>hS', gitsigns.stage_buffer)
    map('n', '<leader>hu', gitsigns.undo_stage_hunk)
    map('n', '<leader>hR', gitsigns.reset_buffer)
    map('n', '<leader>hp', gitsigns.preview_hunk)
    map('n', '<leader>hb', function() gitsigns.blame_line{full=true} end)
    map('n', '<leader>tb', gitsigns.toggle_current_line_blame)
    map('n', '<leader>hd', gitsigns.diffthis)
    map('n', '<leader>hD', function() gitsigns.diffthis('~') end)
    map('n', '<leader>td', gitsigns.toggle_deleted)
    ```

Reply:
> why do we need to use `Gitsigns` in some instances? can't we use `require 'gitsigns'` everywhere?
> 
> like the example from their own repo

either works, we mainly need to update the `next_hunk` call which was deprecated.

It's probably a good idea to keep changes to minimal to avoid confusion, so I'll change it back instead.

Reply:
sounds good to me 👍

Reply:
I left the visual mode mapping as is, since the lua one is awkward and uses vimscript anyway

```lua
map('v', '<leader>hs', function() gitsigns.stage_hunk {vim.fn.line('.'), vim.fn.line('v')} end)
map('v', '<leader>hr', function() gitsigns.reset_hunk {vim.fn.line('.'), vim.fn.line('v')} end)
```